addon/helpers/changeset-get.ts

```diff
@@ -0,0 +1,34 @@
+import Helper from '@ember/component/helper';
+import { observer } from '@ember/object';
+import { ChangesetDef } from 'ember-changeset/types';
+
+const CONTENT = '_content';
+const CHANGES = '_changes';
+
+export default class ChangesetGet extends Helper.extend({
+  invalidate: observer(`changeset.${CONTENT}`, `changeset.${CHANGES}`, function(
```

Comment:
Do you have any documentation on `invalidate`?  Just curious b/c I thought I have seen it as a function call before instead of an observer...

Reply:
The name `invalidate` can be called anything. The key thing here is to call the recompute function, which will re-evaluate the compute method and render the result of that. 

Here is the documentation for the recompute: https://api.emberjs.com/ember/3.9/classes/Helper/methods/recompute?anchor=recompute

Reply:
👍.  Was making sure we weren't inadvertently overriding something